package.json

```diff
@@ -21,8 +21,10 @@
     "react-native-reanimated": "~1.4.0",
     "react-native-safe-area-context": "0.6.0",
     "react-native-screens": "2.0.0-alpha.12",
+    "react-native-vector-icons": "^6.6.0",
```

Comment:
Is react-native-vector-icons used anywhere?

Reply:
Ah, I forgot that these were added to the package.json file due to some experimenting that I was doing. I can all of the changes to that file, as I don't think any of them are needed.